src/utils/datetime.ts

```diff
@@ -38,3 +38,30 @@ export const getEarliestDate = (dates: string[]): number =>
  * @returns {number}
  */
 export const parseDate = (date: string): number => Date.parse(date);
+
+// Formatter for "Today" and "Yesterday" etc
+const relative = new Intl.RelativeTimeFormat('en-GB', { numeric: 'auto' });
+
+// Formatter for weekdays, e.g. "Monday"
+const short = new Intl.DateTimeFormat('en-GB', { weekday: 'long' });
+
+// Formatter for dates, e.g. "Mon, 31 May 2021"
+const long = new Intl.DateTimeFormat('en-GB', {
+  weekday: 'short',
+  day: 'numeric',
+  month: 'short',
+  year: 'numeric',
+});
+
+export const formatDate = (date) => {
```

Comment:
Type of date e.g. `date: Date` and function comment should be added for clarity.